gulpfile.js

```diff
@@ -10,6 +10,7 @@ const git = require('gulp-git-streamed');
 const gulp = require('gulp');
 const gulpif = require('gulp-if');
 const gutil = require('gulp-util');
+const supportedLockFileVersion = [1];
```

Comment:
🍹 Since this is an array, it should probably be called `supportedLockFileVersions`.